packages/costa/package.json

```diff
@@ -23,6 +23,7 @@
     "better-opn": "^1.0.0",
     "debug": "^4.1.1",
     "fs-extra": "^8.1.0",
+    "md5": "^2.3.0",
```

Comment:
The `md5` depends on the [crypt](https://github.com/pvorb/node-crypt/blob/master/crypt.js) which is implemented in JavaScript for browser, I'd prefer do use the Node.js crypto module, see https://nodejs.org/dist/latest-v12.x/docs/api/crypto.html#crypto_crypto_createhash_algorithm_options, it supports the algorithm `md5`, too. And it should be more stable and performant.

Reply:
done